samples/Config.CustomConfigurationProviders.Web/Extensions/ConfigurationProviderKeyExtensions.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Extensions.Configuration;
+
+namespace Microsoft.Extensions.Configuration.Extensions
```

Comment:
I recommend against having *another* `.Extensions` in the namespace. Name this namespace after whatever it is about.